### PR TITLE
Cleanup the kube client tool

### PR DESCRIPTION
**Pull request by WillardHu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

1. The business code should not use `os.Exit` to handle errors.
2. Remove unused `authKubeConfig`

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/common/client/client.go [hunk 1]
```diff
@@ -50,8 +47,7 @@ func InitKubeEdgeClient(config *cloudcoreConfig.KubeAPIConfig, enableImpersonati
 	initOnce.Do(func() {
 		kubeConfig, err := clientcmd.BuildConfigFromFlags(config.Master, config.KubeConfig)
 		if err != nil {
-			klog.Errorf("Failed to build config, err: %v", err)
-			os.Exit(1)
+			panic(fmt.Errorf("failed to build kube config, err: %v", err))
```
Comment (fisherxu) [inline cloud/pkg/common/client/client.go:50]: To print the error stack?

Comment (WillardHu) [reply cloud/pkg/common/client/client.go:50]: Yes, and replace `os.Exit(1)` to `panic(err)`. Printing logs and returning the original error does not help us troubleshoot problems. :-)

**Review by fisherxu — COMMENTED** [review]
/approve

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**